python/mxnet/optimizer/optimizer.py

```diff
@@ -692,18 +692,18 @@ class LBSGD(Optimizer):
     Parameters
     ----------
     momentum : float, optional
-       The momentum value.
-    multi_precision: bool, optional
-       Flag to control the internal precision of the optimizer.
-       ``False`` results in using the same precision as the weights (default),
-       ``True`` makes internal 32-bit copy of the weights and applies gradients
-                in 32-bit precision even if actual weights used in the model have lower precision.`<
-                Turning this on can improve convergence and accuracy when training with float16.
-    warmup_strategy: string ('linear', 'power2', 'sqrt'. , 'lars'   default : 'linear')
-    warmup_epochs: unsigned, default: 5
-    batch_scale:   unsigned, default: 1 (same as batch size*numworkers)
-    updates_per_epoch: updates_per_epoch (default: 32, Default might not reflect true number batches per epoch. Used for warmup.)
-    begin_epoch: unsigned, default 0, starting epoch.
+        The momentum value.
+    multi_precision : bool, optional
+        Flag to control the internal precision of the optimizer.
```

Comment:
Please talk to @Roshrini and let her know what changes you suggest since this [file is getting updated in her PR](https://github.com/apache/incubator-mxnet/pull/13160/files)... might be easier to have hers handle this file's update. Can you describe the need to add the extra space after the parameter name and before the colon?

Reply:
I added four spaces for each new indent (instead of 3). The only change that needs to be in place is replacing ``` with `"` and adding continuation line to "True". Everything else is trying to keep it in sync with other doc strings in the file.